source/tomster.html.erb

```diff
@@ -1,20 +1,29 @@
 ---
 title: "Tomster and Zoey"
+filters: all, tomster, zoey
 ---
 
 <div class="tomster section intro">
   <h1>Tomster and Zoey</h1>
   <p>
-    Tomster was the first of many friendly faces of the Ember project and community. Over the years we've dressed and redressed him for all sorts of fun projects and reasons. With the addition of his sister Zoey, you'll see Ember friends around even more.
+    Tomster was the first of many friendly faces of the Ember project and community. Over the years we"ve dressed and redressed him for all sorts of fun projects and reasons. With the addition of his sister Zoey, you"ll see Ember friends around even more.
   <p>
   <p>
     Tomster is a registered and protected trademark. These images may not be used without specific and explicit consent (which we aim never to hold back unreasonably). If you have questions about using or commissioning a custom Mascot, <a href="/tomster/faq">check out the FAQ</a>.
   </p>
 </div>
 
 <div class="tomster section list">
+  <div id="tomster-filter">
+    <%= active_link_to "All", "/tomster/" %> |
+    <%= active_link_to "Tomster", "/tomster/tomster/" %> |
+    <%= active_link_to "Zoey", "/tomster/zoey/" %> |
+    <%= active_link_to "Meetup", "/tomster/meetup/" %> |
+    <%= active_link_to "Conference", "/tomster/conference/" %> |
+    <%= active_link_to "Other", "/tomster/other/" %>
```

Comment:
Let's change the above links to `/tomster?filter=tomster` and equivalent  Doing that will make life simpler if/when we move this page to Ember (as we don't have to worry about having additional pages to map on the search engine side of things).

**Edit**: hmm, looks like that's not an option we can accommodate with Middleman, carry on ... ;-)